docs/source/contributing/docs-contributions/doc-new-repos.md

```diff
@@ -29,7 +29,7 @@ demo notebooks, if available.
 
 The *Resources* section includes:
 
-![Resources section in `README.md`](static/repo-resources.png "Screenshot of resource list at end of README file")
+![Resources section in `README.md` ](static/repo-resources.png "Screenshot of resource list at end of README file")
```

Comment:
I'm working on this

Reply:
(my WIP which I'm stopping but might be useful to Ely)
Screenshot of resource list at end of README file. In this example, the list contains resources for the ipywidgets project and Project Jupyter. Under ipywidgets there are links to the demo notebook, documentation, issue tracker, and technical support. Under Project Jupyter are links to the Project Jupyter website, the online demo, and documentation.